Bottom: 0f2a7ae83d388bb2a7f1c9c012d593f9e0d2b458
Top:    b523cb06e9592d9d718ebd494baa03dc89c426dd
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2022-12-28 13:45:59 +0100

bsp initial langdale support

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
index 299a0f74bb..ae75c8f2ad 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
+++ b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "{{=machine}}"
 BBFILE_PATTERN_{{=machine}} = "^${LAYERDIR}/"
 BBFILE_PRIORITY_{{=machine}} = "6"
 
-LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus dunfell gatesgarth"
+LAYERSERIES_COMPAT_{{=machine}} = "langdale mickledore nanbield"
